src/index.js

```diff
@@ -0,0 +1,4 @@
+const date = new Date()
+const todayDate = date.toLocaleDateString('de-de');
```

Comment:
It's ok, but you should not force any specific date format. That way will use the German formate. If you leave empty, JS will pick the system format.

---

To simplify, since you probably will not use the date anywhere else:
```js
const todayDate = new Date().toLocaleDateString('de-de');
```

---

Finally, keep consistence in the semicolons (use everywhere, or not use at all): `const date = new Date();`

Reply:
if i leave the format it will display the american format to me for some reason maybe because of my keyboard layout, no idea, will it appear in portuguese layout if i leave it empty?

Reply:
I don't know, test it : )